pkgs/os-specific/linux/cryptodev/default.nix

```diff
@@ -2,28 +2,20 @@
   lib,
   stdenv,
   fetchFromGitHub,
-  fetchpatch,
   kernel ? false,
 }:
 
-stdenv.mkDerivation rec {
-  pname = "cryptodev-linux-1.13";
-  name = "${pname}-${kernel.version}";
+stdenv.mkDerivation (finalAttrs: {
+  pname = "cryptodev-linux-1.14";
```

Comment:
```suggestion
  pname = "cryptodev-linux";
  version = "1.14";
```

Reply:
@SuperSandro2000 Thanks for the review! I've removed the code that appended the kernel version to the derivation name and applied the remaining fixes you suggested. Now it mostly does what other out-of-tree modules do as well.